elife/docker.sls

```diff
@@ -103,3 +103,11 @@ docker-ready:
             - docker-users-in-group
             - docker-scripts
             - docker-scripts-path
+
+# frees disk space from old images/containers/volumes/...
+# older than last X days hours and not in use
+docker-prune-last-days:
+    cmd.run:
+        - name: /usr/local/docker-scripts/docker-prune {{ 24 * pillar.elife.docker.prune_days }}
+        - require:
+            - docker-ready
```

Comment:
Probably doesn't matter when this happens.

Reply:
Must happen after the `docker` binary is available e.g. on installation (in that case it would be almost a no-op)

Reply:
No, it should not matter when it happens; `docker-ready` is mostly for newly created or rebooted VMs so in that case they shouldn't have an almost full disk.